### ISSUE TITLE
Memory "snapshotting"

### ISSUE THREAD

**Issue by dotxnc** [TheLumaio/baslike#1]
Possible syntax
`PUSH`
`POP`

**Comment by dotxnc** [comment]
Use with `RET` register.
```
FNC X
    (Do something with all memory addresses)
    RET 6 (Set RET to value of memory address 6)
END
PUSH
CAL X <1337>
POP
(You now have RET saved and memory back to normal)
```

**Comment by dotxnc** [comment]
fixed 4ee7d2c
